app/views/layouts/application.html.erb

```diff
@@ -4,15 +4,14 @@
     <title> Finden </title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"> 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```

Comment:
I know is not part of your tasks, but please check the extra spaces in this line.